Procfile

```diff
@@ -1 +1,2 @@
 web: gunicorn shielddash.wsgi --log-file -
+api: node_modules/.bin/json-server --watch db.json --port 3009
```

Comment:
This is only useful for localhost, right? We don't need this running on heroku do we?